src/components/Modals/CreateSingleItemModal/ImportStep/ImportStep.tsx

```diff
@@ -245,6 +278,7 @@ export default class ImportStep extends React.PureComponent<Props, State> {
             onDropRejected={this.handleDropRejected}
           />
           {wearablePreviewComponent}
+          <WearablePreview id="wearable-preview" disableBackground disableAutoRotate />
```

Comment:
Are we going to render 2 wearable preview components now? The one above will have the `id` `thumbnail-picker`, which is different from this new one but do we need both?